pinata/doc.md

```diff
@@ -0,0 +1,116 @@
+# How to use Bifrost with Pinata Cloud
+Welcome to the Bifrost documentation for Pinata Cloud! In this guide, we'll show you how to use Bifrost to upload files to Pinata Cloud.
+
+## Overview
+Pinata Cloud is a powerful decentralized cloud storage platform that enables users to securely store and manage digital assets. It allows you to store and access your data on IPFS. Bifrost provides a simple and intuitive way to upload files to Pinata Cloud without having to write complex code.
+
+## Prerequisites
+Before you can start using Bifrost to upload files to Pinata Cloud, you'll need to make sure you have the following:
+- A Pinata Cloud account with API access
+- A Pinata Cloud API key and API secret
```

Comment:
Actually, there are two ways of authenticating Pinata Cloud...we hope to add the second one soon but for now, the only authentication for Pinata supported by Bifrost is via the Pinata JWT.